bcr.cc

```diff
@@ -447,9 +447,9 @@ bcr_route_header(const route_head* route)
     }
 
     if (sout.isEmpty()) {
-      sout = QString("%1,%2,@,0").arg(s1).arg(s1);
+      sout = QString("%1,%2,@,0").arg(s1, s1);
```

Comment:
That's a weird way to write this code anyway. But BCR isn't a format I worry much about.